lib/cli-engine.js

```diff
@@ -402,6 +457,12 @@ CLIEngine.prototype = {
 
         stats = calculateStatsPerRun(results);
 
+        if (options.useCache) {
+            // persist the cache to disk
+            fCache.reconcile();
+        }
+
+        debug("Time: " + (Date.now() - startTime) + "ms");
```

Comment:
Hi @michaelficarra 

Don't get what the issue is `Date.now()` returns a number (same as `(new Date()).getTime()` )